docker/Dockerfile

```diff
@@ -108,30 +126,41 @@ RUN ["/bin/bash", "-c", "if [ -n $INSTALL_CERTS ]; then \
 RUN wget https://github.com/glattercj/vmdb2/releases/download/v1.1/vmdb2 -O /usr/bin/vmdb2 \
   && chmod +x /usr/bin/vmdb2
 
-# do this before installing phenix-apps so minimega package is latest version
-COPY --from=minimega /opt/minimega/lib /tmp/minimega
-RUN  python3 -m pip install /tmp/minimega
-
-ARG APPS_REPO=sandialabs/sceptre-phenix-apps
-ARG APPS_BRANCH=main
-
-RUN python3 -m pip install \
-  --trusted-host pypi.org \
-  --trusted-host files.pythonhosted.org \
-  "git+https://github.com/${APPS_REPO}.git@${APPS_BRANCH}#egg=phenix-apps&subdirectory=src/python"
-
 # needed to build Kali images with phenix
 RUN wget -O kali.deb https://http.kali.org/kali/pool/main/k/kali-archive-keyring/kali-archive-keyring_2022.1_all.deb \
-	&& dpkg -i kali.deb && rm kali.deb
+	&& dpkg -i kali.deb \
+  && rm kali.deb
 
 # used by scorch
 RUN wget -O glow.tgz https://github.com/charmbracelet/glow/releases/download/v1.5.0/glow_1.5.0_Linux_x86_64.tar.gz \
-  && tar -xzf glow.tgz glow && mv glow /usr/local/bin/glow && rm glow.tgz
+  && tar -xzf glow.tgz glow \
+  && mv glow /usr/local/bin/glow \
+  && rm glow.tgz
 
 # used by scorch
 RUN wget -O filebeat.deb https://artifacts.elastic.co/downloads/beats/filebeat/filebeat-7.17.5-amd64.deb \
-	&& dpkg -i filebeat.deb && rm filebeat.deb
+	&& dpkg -i filebeat.deb \
+  && rm filebeat.deb
```

Comment:
If you're going to split this, can you untab the first line so they're both indented the same?